zerver/lib/webhooks/git.py

```diff
@@ -89,6 +89,7 @@ def get_push_commits_event_message(
     commits_data: List[Dict[str, Any]],
     is_truncated: bool = False,
     deleted: bool = False,
+    force_push: Optional[bool] = False,
```

Comment:
Adding an unused parameter as the sole thing a commit doesn't doesn't make sense -- this is not an atomic commit, as the rest of the changes to use it are elsewhere.

Reply:
Rightly said as the changes that use it are somewhere else therefore this commit doesn't makes sense.

I will keep this in mind for the next time.